### PR TITLE
Remove System.out.print in AdminActivityLogPageData and AdminEmailLogPageData #5099 .

### LINKED ISSUE #5099: Remove System.out.print in AdminActivityLogPageData and AdminEmailLogPageData
Both have the same usages: `System.out.print(tokens.length);`.

### PR DESCRIPTION
**Pull request by rajivjhaimin**
Fixes #5099 
Removed System.out.print in AdminActivityLogPageData : Line 271 ,  AdminEmailLogPageData : Line 114.

### CODE REVIEW

**Comment by LowWeiLin** [comment]
@rajivjhaimin Please remove the lines completely instead of commenting it out.

**Comment by rajivjhaimin** [comment]
@LowWeiLin  I have removed the Comments. Kindly have a look.

File: src/main/java/teammates/ui/controller/AdminActivityLogPageData.java [hunk 1]
```diff
@@ -267,7 +267,8 @@ private QueryParameters parseQuery(String query) throws Exception{
         query = query.replaceAll(": ", ":");
         String[] tokens = query.split("\\|", -1); 
        
-        System.out.print(tokens.length);
+        
+
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/AdminActivityLogPageData.java]: Remove these extra blank lines?

File: src/main/java/teammates/ui/controller/AdminEmailLogPageData.java [hunk 2]
```diff
@@ -111,7 +111,7 @@ private QueryParameters parseQuery(String query) throws Exception{
         query = query.replaceAll(": ", ":");
         String[] tokens = query.split("\\|", -1); 
        
-        System.out.print(tokens.length);
+
```
Comment (damithc) [inline src/main/java/teammates/ui/controller/AdminEmailLogPageData.java]: Here too

**Comment by thyageshm** [comment]
@LowWeiLin please merge this pr

**Pull request merged**